### ISSUE TITLE
Shouldn't publishConfig.registry just default to npm's registry if not specified?

### ISSUE THREAD

**Issue by khoomeister** [groupon/nlm#35]
For open source users, this would be the typical default.

**Comment by jkrems** [comment]
I *kind of* like it being explicit, just in case there's ever a bug in `npm` or `yarn` that doesn't force the default to the public registry, even when using a different registry in their `~/.npmrc`. I know that plenty of people were surprised that defaulting behavior exists in the first place.

But I definitely would merge a PR that removes the requirement.

**Comment by khoomeister** [comment]
True, I guess most npm users wouldn't have their own private registry, and for those that do, they've reached expert user status and should be aware of what they're doing :grin:

**Comment by jkrems** [comment]
There are tons of people in EU/Asia that use npm proxies though or people that use a local npm cache. It's not just enterprise use cases. :)
